app/assets/stylesheets/screens/_keyword_details.scss

```
@@ -0,0 +1,6 @@
+.keyword-details {
```

For screens, the body class (see in https://github.com/Goose97/google-scraper-ruby/blob/develop/app/views/layouts/application.html.erb#L13) name must be used:

```suggestion
.keywords.show {
```

We also missed that in https://github.com/Goose97/google-scraper-ruby/blob/develop/app/assets/stylesheets/screens/_keywords.scss#L1 where is should be:

```
.keywords.index {
```

> [!NOTE]
> We could combine all keyword-related screens into one file: `_keywords.scss` then have multiple sections:
> ```
> .keywords.index {}
> 
> .keywords.show {}
> ```

---

Fixed in 9e620782ea3de6556291ff7e0a21faf4485716f2 as Andy suggested